Source/WTF/wtf/CheckedPtr.h

```diff
@@ -212,6 +212,34 @@ inline bool is(const CheckedPtr<ArgType, ArgPtrTraits>& source)
     return is<ExpectedType>(source.get());
 }
 
+template<typename Target, typename Source, typename PtrTraits>
+inline CheckedPtr<match_constness_t<Source, Target>> uncheckedDowncast(CheckedPtr<Source, PtrTraits> source)
```

Comment:
So we need two variants one which takes `const CheckedPtr<Source, PtrTraits>&` which just calls `get` and another which takes `CheckedPtr<Source, PtrTraits>&&` and does `WTFMove`.